source/CoordinateConversion/CoordinateConversionLibrary/ViewModels/EditOutputCoordinateViewModel.cs

```diff
@@ -37,16 +37,6 @@ public EditOutputCoordinateViewModel()
                                                                     Properties.Resources.CategoryListUSNG,
                                                                     Properties.Resources.CategoryListUTM };
             }
-            else
-            {
-                CategoryList = new ObservableCollection<string>() { Properties.Resources.CategoryListDD,
```

Comment:
This is the only change that looks like it might cause a problem. The code is repeated and definitely should be fixed but this change looks like it won't work when Title != "Pro" 

@topowright @kgonzago - how would you like me to address? Just merge and then fix?

Reply:
@csmoore Yes. Please fix this issue and then merge.